binstar_client/utils/detect.py

```diff
@@ -76,6 +78,13 @@ def is_r(filename):
     else:
         log.debug("This not is an R package (expected .tgz, .tar.gz).")
 
+def is_cluster_profile(filename):
```

Comment:
I do that in the `inspect_profile_package` function:  https://github.com/Anaconda-Server/anaconda-client/blob/feature/profile_upload/binstar_client/inspect_package/profile.py#L9.  Butt we are also going to be adding another key/value pair indicating api version and that it's an anaconda cluster profile